File: Web/src/Entity/Tutorial.php
<?php

namespace App\Entity;

use App\Repository\TutorialRepository;
use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Validator\Constraints as Assert;

/**
 * @ORM\Entity(repositoryClass=TutorialRepository::class)
 */
class Tutorial
{
    /**
     * @ORM\Id
     * @ORM\GeneratedValue
     * @ORM\Column(type="integer")
     */
    private $id;

    /*
    /**
     * @ORM\Column(type="string",length=255)
     * @Assert\NotBlank(message="You must select a video for your tutorial !")
     */
    /*private $video;*/


    /**
     * @ORM\Column(type="datetime")
     * @Assert\NotBlank(message="You must select a Date for your tutorial !")
     */
    private $dateTuto;

    /**
     * @ORM\Column(type="string",length=255)
     * @Assert\NotBlank(message="You must select a Category for your tutorial !")
     */
    private $category;

    /**
     * @ORM\Column(type="string",length=255)
     * @Assert\NotBlank(message="Title can not be empty !")
     */
    private $titre;

    /**
     * @ORM\Column(type="string",length=255)
     * @Assert\NotBlank(message="Description can not be empty !")
     */
    private $description;

    /**
     * @ORM\Column(type="float")
     * @Assert\NotBlank(message="Prix can not be empty !")
     */
    private $prix;

    
    public function getId()
    {
        return $this->id;
    }


    public function setId($id): void
    {
        $this->id = $id;
    }


    public function getVideo()
    {
        return $this->video;
    }

    public function setVideo($video): void
    {
        $this->video = $video;
    }


    public function getDateTuto()
    {
        return $this->dateTuto;
    }


    public function setDateTuto($dateTuto): void
    {
        $this->dateTuto = $dateTuto;
    }


    public function getCategory()
    {
        return $this->category;
    }


    public function setCategory($category): void
    {
        $this->category = $category;
    }


    public function getTitre()
    {
        return $this->titre;
    }


    public function setTitre($titre): void
    {
        $this->titre = $titre;
    }


    public function getDescription()
    {
        return $this->description;
    }


    public function setDescription($description): void
    {
        $this->description = $description;
    }


    public function getPrix()
    {
        return $this->prix;
    }


    public function setPrix($prix): void
    {
        $this->prix = $prix;
    }
    
    
    
    
    


   
    
    
}

File: Mobile/src/com/esprit/app/entity/Tutorial.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.esprit.app.entity;


public class Tutorial {
    private int id;
    private String video;
    private String image;
    private String dateTuto;
    private String category;
    private String titre;
    private String description;
    private double prix;

    public Tutorial(int id, String video, String image, String dateTuto, String category, String titre, String description, double prix) {
        this.id = id;
        this.video = video;
        this.image = image;
        this.dateTuto = dateTuto;
        this.category = category;
        this.titre = titre;
        this.description = description;
        this.prix = prix;
    }

    public Tutorial(String video, String image, String dateTuto, String category, String titre, String description, double prix) {
        this.video = video;
        this.image = image;
        this.dateTuto = dateTuto;
        this.category = category;
        this.titre = titre;
        this.description = description;
        this.prix = prix;
    }

    public Tutorial() {
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getVideo() {
        return video;
    }

    public void setVideo(String video) {
        this.video = video;
    }

    public String getImage() {
        return image;
    }

    public void setImage(String image) {
        this.image = image;
    }

    public String getDateTuto() {
        return dateTuto;
    }

    public void setDateTuto(String dateTuto) {
        this.dateTuto = dateTuto;
    }

    public String getCategory() {
        return category;
    }

    public void setCategory(String category) {
        this.category = category;
    }

    public String getTitre() {
        return titre;
    }

    public void setTitre(String titre) {
        this.titre = titre;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public double getPrix() {
        return prix;
    }

    public void setPrix(double prix) {
        this.prix = prix;
    }
    
    @Override
    public String toString() {
        return "Tutorial{" + "video=" + video + ", image=" + image + ", dateTuto=" + dateTuto + ", category=" + category + ", titre=" + titre + ", description=" + description + ", prix=" + prix + '}';
    }
    
}